css/style.css

```diff
@@ -149,6 +149,41 @@ html {
   font-size: 2rem;
 }
 
+/******************************/
+/* GENERAL STYLES FOR HEADINGS AND TEXT */
+/******************************/
+
+.heading-secondary {
+  font-family: "Yeseva One", sans-serif;
+  font-size: 3.6rem;
+  font-weight: 400;
+  margin-bottom: 1.6rem;
+}
+
+.heading-light {
+  color: #FFFFFF;
+}
+
+.heading-dark {
+  color: #cc9900;
+}
+
+.main-description {
+  font-family: 'Lato', sans-serif;
+  font-size: 1.6rem;
+  font-weight: 400;
+  line-height: 2.4rem;
+  margin-bottom: 4.8rem;
+}
+
+.text-style-dark {
+  color: #4c4c4c;
+}
+
+.text-style-light {
+  color: #FFFFFF;
```

Comment:
Usually single style is used (only capitals or only non-capitals)
```css
color: #fff;
```

Reply:
Yes, I remembered that. I did not notice how I used a large register.